src/main/java/ca/team3161/lib/robot/BlinkinLEDController.java

```diff
@@ -0,0 +1,158 @@
+package ca.team3161.lib.robot;
+
+import java.util.concurrent.TimeUnit; 
+
+import ca.team3161.lib.robot.subsystem.RepeatingPooledSubsystem;
+import edu.wpi.first.wpilibj.Spark;
+
+/**
+ * A class wrapping around a PWM Spark to control a REV Blinkin
```

Comment:
This probably shouldn't mention the `PWM Spark` or other internal implementation details.